UX-STANDARDS.md

```diff
@@ -0,0 +1,11 @@
+# UX Standards
+
+These standards include some basic user experience guidelines we align to on the marketing website, docs, and the registry.
+
+- Every page should have exactly one primary action.
+- Use descriptive text for button labels and links, and avoid labels such as "Learn more".
+- Use unique labels for distrinct actions.
+- Use existing components over creating new components unless you identify a need that is not addressed by the current components.
+- Use line lengths between 45 – 75 characters.
+- Links should take you to another page, and buttons should take an action on the page.
+- When possible links should look like links, and buttons should look like buttons.
```

Comment:
I like that we're making a distinction between links and buttons and both their individual purposes, as well as noting that the two should look different. Since buttons should always take an action on the page, I'm assuming links would primarily be within the context of the text on a page?

Reply:
yes! i think thats a good thing to be clear about here